cmake/FindApMon.cmake

```diff
@@ -0,0 +1,45 @@
+# - Try to find ApMon include dirs and libraries
+# Author: Barthelemy von Haller
+# Author: Adam Wegrzynek
+#
+# This script will set the following variables:
+#  ApMon_FOUND - System has ApMon
+#  ApMon_INCLUDE_DIRS - The ApMon include directories
+#  ApMon_LIBRARIES - The libraries needed to use ApMon
+#  ApMon_DEFINITIONS - Compiler switches required for using ApMon
+# This script can use the following variables:
+#  APMON_ROOT - Installation root to tell this module where to look.
+# This script defines following targets:
+#  ApMon::ApMon
+
+# Init
+include(FindPackageHandleStandardArgs)
+
+# find includes
+find_path(ApMon_INCLUDE_DIR ApMon.h
+        HINTS ${APMON_ROOT}/include
+        PATHS /usr/local/include)
```

Comment:
Not sure if this does what you want. Is `${APMON_ROOT}` always defined? If not, you will hint `/include`. An idea:

```cmake
find_path(ApMon_INCLUDE_DIR ApMon.h
        HINTS ${APMON_ROOT} /usr/local
        PATH_SUFFIXES include)
```

I would guess `/usr/local` is in the default system search paths already. If yes and you drop it, test if the command can work with an empty `HINTS` parameter in case `APMON_ROOT` is empty.

Reply:
Thanks for reviewing! Yep this is definitely good idea